ddocs/medic/views/doc_summaries_by_id/map.js

```diff
@@ -73,15 +73,16 @@ function(doc) {
       lineage: getLineage(doc.contact && doc.contact.parent),
       subject: getSubject(doc)
     });
-  } else if (doc.type === 'clinic' ||
+  } else if (doc.type === 'contact' ||
+      doc.type === 'clinic' ||
       doc.type === 'district_hospital' ||
       doc.type === 'health_center' ||
       doc.type === 'person') { // contact
     emit(doc._id, {
       _rev: doc._rev,
       name: doc.name || doc.phone,
       phone: doc.phone,
-      type: doc.type,
+      type: doc.contact_type || doc.type,
```

Comment:
I think this emit needs to change to include `type` and `contact_type`, as this summary will be passed to LiveList sorting for online users, which requires knowing both type and contact_type.

Reply:
Right. Done.